packages/now-client/src/utils/hashes.ts

```diff
@@ -42,22 +43,23 @@ export const mapToObject = (
  * @return {Map}
  */
 async function hashes(files: string[]): Promise<Map<string, DeploymentFile>> {
-  const map = new Map();
+  const map = new Map<string, DeploymentFile>();
   const semaphore = new Sema(100);
 
   await Promise.all(
     files.map(
       async (name: string): Promise<void> => {
         await semaphore.acquire();
         const data = await fs.readFile(name);
+        const { mode } = await fs.stat(name);
```

Comment:
Should this be `lstat()` instead? I'm not sure if we support symlinks in file uploads…

Reply:
I think `fs.readFile` is going to follow the symlink so there would need to be a change there too, right?

Reply:
Ya, proper symlink support requires some API changes it seems. Using `fs.stat()` here in this case is correct.